pkg/reconciler/knativeserving/ingress/ingress_test.go

```diff
@@ -100,16 +101,16 @@ func TestAppendInstalledIngresses(t *testing.T) {
 			},
 			Status: servingv1alpha1.KnativeServingStatus{},
 		},
-		expected:             true,
-		expectedResourcesNum: numberIngressResource,
+		expectedIngressPath: os.Getenv(common.KoEnvKey) + "/ingress/0.18",
+		expectedErr:         nil,
 	}}
```

Comment:
Do we want to add one more test case `"Unavailable ingresses"` with invalid version?

Reply:
done